packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -24,18 +21,19 @@ export class InserterMenu extends Component {
 	constructor() {
 		super( ...arguments );
 
+		this.onClose = this.onClose.bind( this );
 		this.onLayout = this.onLayout.bind( this );
 		this.state = {
 			numberOfColumns: this.calculateNumberOfColumns(),
 		};
 	}
 
 	componentDidMount() {
-		this.onOpen();
+		this.props.showInsertionPoint();
```

Comment:
In the future refactoring, these components to hooks may increase reusability between mobile and web.
E.g: We can have specific behaviors as hooks like useShowInserPointHook ( hook that trigger the show insert point on the mount and removes it on unmount ) and then the difference between web mobile could simply be the hooks (behaviors) that each one uses.

Reply:
Hooks ftw 🎉